pre-release v1.1.43 (#1029)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-cloud/dbt-cloud-changelog.md

description: "Changelog for the dbt Cloud application"
---

## dbt Cloud v1.1.43 (January 19th, 2022)

> We have made some improvements this time around and added a [new metric layer page](https://docs.getdbt.com/docs/dbt-cloud/using-dbt-cloud/cloud-metrics-layer) to the product docs to help you begin thinking about the metrics layer in dbt Cloud.

Reviewer: ```suggestion
We have made some improvements this time around (see the highlights below) and added a [new metric layer page](https://docs.getdbt.com/docs/dbt-cloud/using-dbt-cloud/cloud-metrics-layer) to docs.getdbt.com to help you begin thinking about the metrics layer in dbt Cloud.
```

> We have made some improvements this time around (see the highlights below) and added a [new metric layer page](https://docs.getdbt.com/docs/dbt-cloud/using-dbt-cloud/cloud-metrics-layer) to docs.getdbt.com to help you begin thinking about the metrics layer in dbt Cloud.

Reviewer: ```suggestion
We have made some improvements that include the new autocomplete snippets for sql and yaml files in the IDE, which are available for use now! We also added a [new metric layer page](https://docs.getdbt.com/docs/dbt-cloud/using-dbt-cloud/cloud-metrics-layer) to docs.getdbt.com to help you begin thinking about the metrics layer in dbt Cloud.
```

> We have made some improvements that include the new autocomplete snippets for sql and yaml files in the IDE, which are available for use now! We also added a [new metric layer page](https://docs.getdbt.com/docs/dbt-cloud/using-dbt-cloud/cloud-metrics-layer) to docs.getdbt.com to help you begin thinking about the metrics layer in dbt Cloud.

Reviewer: ```suggestion
Some noteworthy improvements include autocomplete snippets for sql and yaml files in the IDE, which are available for use now! We also added a [new metric layer page](https://docs.getdbt.com/docs/dbt-cloud/using-dbt-cloud/cloud-metrics-layer) to docs.getdbt.com to help you begin thinking about the metrics layer in dbt Cloud.
```

#### Performance improvements and enhancements

* Branch names now default to "main" instead of "master" in new managed and unmanaged Git repositories.

> * Fixed the stale status of run steps on canceled runs.

Reviewer: ```suggestion
* Update IDE autocomplete snippets.
```

> 

Reviewer: ```suggestion
```
